### ISSUE TITLE
markdown-it-emoji plugin

### ISSUE THREAD

**Issue by neumayr** [digitalmoksha/motion-markdown-it-plugins#3]
Hey @digitalmoksha 👋 

Awesome port 👏 I would like to ask if you are interested in a `markdown-it-emoji` plugin?

📦 https://github.com/markdown-it/markdown-it-emoji

I didn't checked all the implementation in detail, but i guess i could volunteer. 🤓

**Comment by digitalmoksha** [comment]
Thanks @neumayr !

I have been wanting to include the emoji plugin.  I would happily accept a pull request.

I would have one requirement - for now, I want the ports kept one to one as much as possible.  This makes it much easier to incorporate changes to the js versions.  You can see how I structured the other plugins as a starting point.

**Comment by neumayr** [comment]
Hi @digitalmoksha, I've started some part with #5 and wanted to ask if we could collaborate on this, I'm failing on the js to ruby translation part.

**Comment by digitalmoksha** [comment]
Yeah, sounds good!  I'll try to peel off some time and take a look at what you've got...

**Comment by digitalmoksha** [comment]
@neumayr I took a first stab on plugin, building on what you already did.  This one is a little more complex than the other plugins.   Moved the data files to Ruby, stubbed some of the functions, and pulled in some of the regexp that are needed.

Will try to do a little more if I can tonight, otherwise it might be several days before I can get back to it.

wdyt, you want to take the next crack at it?
